### DISCUSSION TITLE
how to profiling clickhouse?

### DISCUSSION THREAD

**Discussion by zh-jn** [ClickHouse/ClickHouse#28008] [Q&A] [answered]
Hi all,
  Is there have any suggestions about profiling the clickhouse?

**Comment by morozovsk** [comment]
change in [config logger level](https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings/#server_configuration_parameters-logger) to trace/debug/information.

**Comment by alexey-milovidov** [comment, marked as answer]
https://clickhouse.com/docs/en/operations/optimizing-performance/sampling-query-profiler/
